h/static/scripts/test/root-thread-test.js

```diff
@@ -179,6 +181,53 @@ describe('rootThread', function () {
     ]);
   });
 
+  describe('when the thread filter query is set', function () {
+    it('generates a thread filter function to filter out annotations', function () {
```

Comment:
The name of this test differs from what the test actually checks. Here this test generates a filter function which only _matches_ (not filters out) annotations. Same with the test below for notes.